src/common/ffile.cpp

```diff
@@ -336,7 +336,7 @@ bool wxTempFFile::Open(const wxString& strName)
     mode_t mode;
 
     wxStructStat st;
-    if ( stat( (const char*) m_strName.fn_str(), &st) == 0 )
+    if ( wxCRT_Stat( (const char*) m_strName.fn_str(), &st) == 0 )
```

Comment:
As long as we're doing this anyhow, we should use our wrappers functions instead of `wxCRT_xxx` macros directly, i.e.
```suggestion
    if ( wxStat(m_strName, &st) == 0 )
```
Otherwise there is really not much point, is there?